search/base.py

```diff
@@ -44,9 +44,9 @@ class MockedESTestCase(TestCase):
     @classmethod
     def setUpClass(cls):
         cls.patchers = []
-        for name, val in indexing_api.__dict__.items():
-            if callable(val):
-                cls.patchers.append(patch('search.indexing_api.{0}'.format(name), autospec=True))
+        for name, val in tasks.__dict__.items():
+            if callable(val) and name.startswith("_"):
```

Comment:
In `search.tasks` we are basically just importing functions from `search.indexing_api` with the same name. They are prefixed with `_` to prevent a name collision